viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,16 +2,32 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
+uniform sampler2D colorDataTexture;
+
 const int RenderTypeColor = 1;
 const int RenderTypeNormal = 2;
 const int RenderTypeTreeIndex = 3;
 const int RenderTypePackColorAndNormal = 4;
+const float dataTextureWidth = 2048.0;
+const float dataTextureHeight = 2048.0;
```

Comment:
This should be provided from JS-side of things. We do not currently have this information available, but the shaders should at least be ready for it.

Reply:
Agreed, but let us keep them here for now and update the shaders once we have this ready.